external-crates/move/move-compiler/transactional-tests/tests/dependencies/public_package_different_packages.exp

```diff
@@ -0,0 +1,10 @@
+processed 3 tasks
+
+task 2 'publish'. lines 11-17:
+Error: Unable to publish module '00000000000000000000000000000042::Y'. Got VMError: {
+    major_status: LOOKUP_FAILED,
+    sub_status: None,
+    location: 0x42::Y,
+    indices: [(FunctionHandle, 1)],
+    offsets: [],
+}
```

Comment:
This does not feel like the correct error. What's going on here?

Reply:
Discussed offline -- seems to be how the VM models things right now.